docs/troubleshoot/troubleshoot-apiml-error-codes.md

```diff
@@ -659,7 +659,7 @@ The following error message codes may appear on logs or API responses. Use the f
 
   **Action:**
 
-  Use a valid body in the request. Format of a message: `{validity: int , scopes: [string]}`.
+  Use a valid body in the request. Format of a message: {validity: int , scopes: [string]}.
```

Comment:
```suggestion
  Use a valid body in the request. Format of a message: `{validity: int , scopes: [string]}`.
```

The build job is failing because in Docusaurus v3 the Markdown files are parsed as MDX and braces must be escaped:

![image](https://github.com/zowe/docs-site/assets/22344007/b80bb096-72cf-4d64-ae57-09ab1ee1bdbf)